plugins/zookeeper.yaml

```diff
@@ -3,11 +3,12 @@ version: 0.0.2
 title: Apache Zookeeper
 description: Log parser for Apache Zookeeper
 parameters:
-  - name: file_path
-    label: Zookeeper logs path
+  - name: file_log_path
     description: The absolute path to the Zookeeper logs
-    type: string
-    default: "/home/kafka/kafka/logs/zookeeper.log"
+    type: strings
+    default: 
+    - "/opt/zookeeper/logs/zookeeper-*.out"
+    - "/var/log/zookeeper/zookeeper.log"
```

Comment:
Are these the same default values as ops-agent? Generally, if zookeeper is writing logs to one, it will not be writing to the other. 

In my opinion, we keep the default the same.

Reply:
yeah, these are the default paths that are looked for in the [ops-agent](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/apps/zookeeper.go#L136-L138) .
  When you say keep the default the same, you are talking about `"/home/kafka/kafka/logs/zookeeper.log"` and then the user would have to specify one ops-agent path or the other?